### ISSUE TITLE
[issue]: Casper Excalibur Boot problems and 1.0.55 about...

### ISSUE THREAD

**Issue by NorthLight07** [ventoy/Ventoy#1794]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

New versions and 1.0.55

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### BIOS Mode

UEFI Mode

### Partition Style

GPT

### Disk Capacity

32GB

### Disk Manufacturer

Lexar 

### What happened?

Computer: Casper Excalibur G770-1030
BIOS:CP171
UEFI BIOS(GPT)
Secure boot: Disabled

Hello ventoy family ^^ 

I wanted to share a problem and its solution. 

I wanted to install Fedora as dual boot on my computer and wrote it as GPT on my USB memory, but the BIOS did not see it, I turned off Secure boot to fix it, etc. But it still did not appear in boot. Menu. But I tried the old 1.0.55 version and it worked. I wanted to share this problem with you.

Computer(Casper) often encountered Boot problems.

**Comment by adib1462** [comment]
we tried on GPT,MBR Secure boot on/off... and the pc didnt see the usb
we spend 1 week for this
in the end we try the old version(1.0.55) and worked :smile:

**Comment by steve6375** [comment]
what partition type and secure boot option did you use with the old version?
Was it the exact same USB drive?
If you use the same drive and same options on 1.0.79 now (not before but now) - does it still fail to see the drive?

**Comment by adib1462** [comment]
yes everything same
only ventoy ver its different
its still fail to see the drive 
we tried arround 1 week and always failed to see the usb

**Comment by steve6375** [comment]
what partition type and secure boot option did you use with the old version?
If you use the same drive and same options on 1.0.79 now (**not before but now**) - does it still fail to see the drive?

**Comment by NorthLight07** [comment]
No, ventoy does not see the BIOS at all in new versions, in fact, when we print GPT with other programs (rufus, YUMI, etc.), it does not see it again, the BIOS only sees Windows.

only working Ventoy 1.0.55 my computer.

**Comment by NorthLight07** [comment]
> No, ventoy does not see the BIOS at all in new versions, in fact, when we print GPT with other programs (rufus, YUMI, etc.), it does not see it again, the BIOS only sees Windows.
> 
> only working in regular Ventoy 1.0.55.

**Comment by asandikci** [comment]
@fixin0 any update on this?

**Comment by fixin0** [comment]
> [@fixin0](https://github.com/fixin0) any update on this?

To be honest, I'm still using the version I mentioned. :)

**Comment by ventoy** [comment]
Clean old issues.
Please test with latest Ventoy and make a new issue if there is still problem.
